test/e2e/dynamic_provisioning.go

```diff
@@ -75,7 +87,7 @@ func (t *dynamicProvisioningTestSuite) defineTests(isMultiZone bool) {
 	})
 
 	testDriver = driver.InitAzureDiskDriver()
-	It(fmt.Sprintf("should create a volume on demand"), func() {
+	ginkgo.It("should create a volume on demand", func() {
```

Comment:
this case is already covered by below case `should create a volume on demand with mount options`, we could remove this case to save time